lib/src/jpegr.cpp

```diff
@@ -1479,16 +1501,17 @@ uhdr_error_info_t JpegR::applyGainMap(uhdr_raw_image_t* sdr_intent, uhdr_raw_ima
 #endif
           }
 
-          rgb_hdr = rgb_hdr / display_boost;
           size_t pixel_idx = x + y * dest->stride[UHDR_PLANE_PACKED];
 
           switch (output_ct) {
             case UHDR_CT_LINEAR: {
+              rgb_hdr = rgb_hdr / gainmap_metadata->hdr_capacity_max;
```

Comment:
I feel with the logic the input `max_display_boost` will become unused in this pass. Also the logic in the OpenGL pass isn't updated

Reply:
Thank you. I have addressed this.